L1Trigger/Phase2L1ParticleFlow/src/newfirmware/dataformats/datatypes.h

```diff
@@ -143,6 +145,7 @@ namespace l1ct {
     constexpr float Z0_LSB = 0.05;
     constexpr float DXY_LSB = 0.05;
     constexpr float PUPPIW_LSB = 1.0 / 256;
+    constexpr float ISO_LSB = 1.0 / 100;
```

Comment:
RELISO_LSB

That said, firmware-wise it may be better to store abs iso to avoid the division, and if later you want to apply a rel-iso cut you can do it cheaply by multiplying pT by a contant.